pkg/services/apiserver/builder/request_handler.go

```diff
@@ -13,7 +13,7 @@ type requestHandler struct {
 	router *mux.Router
 }
 
-func getAPIHandler(delegateHandler http.Handler, restConfig *restclient.Config, builders []APIGroupBuilder) (http.Handler, error) {
+func GetCustomRoutesHandler(delegateHandler http.Handler, restConfig *restclient.Config, builders []APIGroupBuilder) (http.Handler, error) {
```

Comment:
Needs to be public if we would like to copy the default build handler chain and insert our various injection points in it wherever we like.